### PR TITLE
remove some unused image versions

### PR DESCRIPTION
**Pull request by alexeldeib**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
none
```

### CODE REVIEW

**Comment by alexeldeib** [comment]
from rp
```
{{- else if eq .component "sgx-plugin" -}}
0.5
{{- else if eq .component "sgx-attestation" -}}
3.1
{{- else if eq .component "sgx-webhook" -}}
1.0
```

**Comment by alexeldeib** [comment]
dashboard - unused in any supported versions
azure-cloud-node-manager - unused or used in unsupported versions
sgx - unused

**Review by ganeshkumar5699 — APPROVED** [review]

File: vhdbuilder/packer/components.json [hunk 1]
```diff
@@ -10,12 +10,9 @@
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/dashboard:*",
       "amd64OnlyVersions": [
-        "v2.0.0",
         "v2.0.1"
       ],
-      "multiArchVersions": [
-        "v2.4.0"
-      ]
+      "multiArchVersions": []
```
Comment (ganeshkumar5699) [inline vhdbuilder/packer/components.json:15]: missing ","?

**Comment by alexeldeib** [comment]
pause - converted fully to 3.6 for all k8s versions

File: vhdbuilder/packer/components.json (hunk 1 above)
Comment (alexeldeib) [reply vhdbuilder/packer/components.json:15]: think I got it

**Pull request merged**